exovetter/lpp.py

```diff
@@ -387,7 +387,8 @@ class Loadmap:
         If URL is provided, it will be cached using :ref:`astropy:utils-data`
 
     """
-    builtin_mat = 'https://sourceforge.net/p/lpptransitlikemetric/code/HEAD/tree/data/maps/combMapDR25AugustMapDV_6574.mat?format=raw'  # noqa: E501
+    builtin_mat = 'https://stsci.box.com/s/s2jup12rcjn05qll598h33bwycjeym8x'
```

Comment:
This link doesn't work. It is not the static link to the raw data. I am just going to include the file as package data.

Reply:
I take it back. GitHub blocked the data file because it exceeded 100 MB. I uploaded it to another space on Box that allows proper download. We can change it back (again) after you fix this Box link, as I don't have enough permission to do anything useful with it.